src/QueryReflection/ReflectionCache.php

```diff
@@ -334,14 +331,7 @@ public function putResultType(string $queryString, int $fetchType, ?Type $result
             $this->changes[$queryString]['result'][$fetchType] = $this->records[$queryString]['result'][$fetchType] = $resultType;
             $this->cacheIsDirty = true;
         }
-    }
-
-    private function isPHPStormSingleFileRun(): bool
```

Comment:
dropping the phpstorm workaround again. the workaround also made the terminal integration not persisting the cache anymore (no matter fullproject scan or not)